common/middleware/instrument_test.go

```diff
@@ -0,0 +1,29 @@
+package middleware_test
+
+import (
+	"testing"
+
+	"github.com/weaveworks/scope/common/middleware"
+)
+
+func TestMakeLabelValue(t *testing.T) {
+	for input, want := range map[string]string{
+		"/":                      "root", // special case
+		"//":                     "root", // unintended consequence of special case
+		"a":                      "a",
+		"/foo":                   "foo",
+		"foo/":                   "foo",
+		"/foo/":                  "foo",
+		"/foo/bar":               "foo_bar",
+		"foo/bar/":               "foo_bar",
+		"/foo/bar/":              "foo_bar",
+		"/foo/{orgName}/Bar":     "foo_orgname_bar",
+		"/foo/{org_name}/Bar":    "foo_org_name_bar",
+		"/foo/{org__name}/Bar":   "foo_org_name_bar",
+		"/foo/{org___name}/_Bar": "foo_org_name_bar",
```

Comment:
Maybe also add a case or two for other invalid values? e.g. `/foo.bar/baz.qux/` -> `foo_bar_baz_qux`